script/releases/v1.0.0-slashing/3-pause.s.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.12;
+
+import "../Env.sol";
+
+import {MultisigBuilder} from "zeus-templates/templates/MultisigBuilder.sol";
+
+/**
+ * Purpose: Enqueue a transaction which immediately sets `EigenPodManager.PAUSED_START_CHECKPOINT=true` 
+ */
+contract Pause is MultisigBuilder {
+    using Env for *;
+
+    function options() internal override view returns (MultisigOptions memory) {
+        return MultisigOptions(
+            Env.pauserMultisig(),
+            Operation.Call
+        );
+    }
+
+    function runAsMultisig() internal virtual override {
+        // TODO: set: 
+        //    pause EigenPodManager
```

Comment:
Can we also pause programmatic incentives as that has not been updated. 

`PAUSED_REWARD_ALL_STAKERS_AND_OPERATORS` in the RC

Reply:
Nvm, going to hard fork